R/PipeOpTuneThreshold.R

```diff
@@ -143,7 +143,12 @@ PipeOpTuneThreshold = R6Class("PipeOpTuneThreshold",
     },
     .task_to_prediction = function(input) {
       prob = as.matrix(input$data(cols = input$feature_names))
-      colnames(prob) = unlist(input$levels())
+      # setting the column names the following way is safer
+      nms = map_chr(strsplit(colnames(prob), "\\."), function(x) x[length(x)])
```

Comment:
maybe breaks when factor level has a period?

Reply:
Better way: use input$levels(input$target_name), generate putative colnames from that, then compare to given col names. the assert is good though.